charts/nginx/values.yaml

```diff
@@ -32,14 +32,17 @@ resources: {}
 #   cpu: 100m
 #   memory: 128Mi
 
+# dict of annotations to add to the ingress controller
```

Comment:
Would like to make sure that we have an example either here in the helm chart or in documentation somewhere for customers that aren't as familiar with Go for what a dict looks like in the language to lessen questions to CRE or in the forum